contracts/eosio.wrap/src/eosio.wrap.cpp

```diff
@@ -10,7 +10,18 @@ void wrap::exec( ignore<name>, ignore<transaction> ) {
 
    require_auth( executer );
 
-   send_deferred( (uint128_t(executer.value) << 64) | (uint64_t)current_time_point().time_since_epoch().count(), executer, _ds.pos(), _ds.remaining() );
+   transaction_header trx_header;
+   std::vector<action> context_free_actions;
+   std::vector<action> actions;
+   _ds >> trx_header;
+   check( trx_header.expiration >= current_time_point(), "transaction expired" );
```

Comment:
There is no need for the expiration check for the `wrap::exec` action.

Reply:
done